lib/customer.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'menu'
+
+class Customer
+
+  attr_accessor :menu, :new_order, :menu, :order, :total_items, :order_total
+
+  def initialize
+    @menu = Menu.new
+    @order = {}
+    @order_total = 0
+    @total_items = 0
+  end
+
+  def receipt
+    order.map do |dish, quantity|
+      "#{dish} x#{quantity} - £#{menu.menu_list[dish] * quantity}"
+    end.join(", ")
+  end
+
+  def display_menu
+    menu.menu_list.each { |key, value| puts "#{key} - £#{value}" }
+  end
+
+  def select_dish(dish, quantity)
+      fail 'Dish not on menu' unless menu.menu_list.has_key?(dish)
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`.